test/test-readability.js

```diff
@@ -176,6 +176,10 @@ function runTestsWithItems(label, domGenerationFn, source, expectedContent, expe
     expectedMetadata.dir && it("should extract expected direction", function() {
       expect(result.dir).eql(expectedMetadata.dir);
     });
+
+    expectedMetadata.lang && it("should extract expected direction", function() {
```

Comment:
We should update the message here to "should extract expected lang"

Reply:
Sorry, I don't know how I missed that, should be good now! :)